addons/product_images/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_product_get_pic_barcode_manager,product.images.get.pic.barcode manager,model_product_images_get_pic_barcode,base.group_system,1,1,1,0

```

Comment:
Is it on purpose that only admins can use this feature?

Reply:
Yes, we thought with @pierre-pde that it's an action that they would not use daily and we wanted to avoid users use it without limits since the call may be paying.

Reply:
Then it should be mentioned in the spec.

Reply:
@pierre-pde has mentioned that in the spec.